apps/src/templates/rubrics/rubricShapes.jsx

```diff
@@ -41,3 +41,9 @@ export const submittedEvaluationShape = PropTypes.shape({
   feedback: PropTypes.string,
   understanding: PropTypes.number,
 });
+
+export const aiEvaluationShape = PropTypes.shape({
+  learning_goal_id: PropTypes.number,
+  understanding: PropTypes.number,
+  ai_confidence: PropTypes.number,
```

Comment:
should any of these be required?

Reply:
learning_goal_id should be and I think understanding probably should be as well. We do gracefully handle confidence not being there, so that probably doesn't need to be